addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,77 @@
+If a device is removed from the account the respective openHAB Thing will go OFFLINE.
+
+## Channels
+
+### Device
+
+The following channels are available (if supported by the device):
+
+| Channel ID         | Type     | Description                                                                                                                                 |
+|--------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| batteryStatus      | String   | Current battery status (Charging, Charged, Unknown)                                                                                         |
+| batteryLevel       | Number   | Battery charge in %                                                                                                                         |
+| findMyPhone        | Switch   | Triggers the ["Find my phone"](https://support.apple.com/explore/find-my-iphone-ipad-mac-watch) functionality of the device (if available). |
+| location           | Location | Location of the device.                                                                                                                     |
+| locationAccuracy   | Number   | Accuracy of the last position report.                                                                                                       |
+| locationLastUpdate | DateTime | Timestamp of the last location update.                                                                                                      |
+
+## Full Example
+
+### icloud.things
+
+```php
+Bridge icloud:account:myaccount [appleId="abc@xyz.tld", password="secure", refreshTimeInMinutes=5]
+{
+    Thing device myiPhone8 "iPhone 8" @ "World" [deviceId="VIRG9FsrvXfE90ewVBA1H5swtwEQePdXVjHq3Si6pdJY2Cjro8QlreHYVGSUzuWV"]
+}
+```
+
+The "label" @ "location" part is optional (as always).
+
+### icloud.items
+
+```php
+Group    iCloud_Group
+
+String   iPhone_BatteryStatus             "Battery Status [%s %%]"             <battery> (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:batteryStatus"}
+Number   iPhone_BatteryLevel              "Battery Level [%.0f]"               <battery> (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:batteryLevel"}
+Switch   iPhone_FindMyPhone               "Trigger Find My iPhone"                       (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:findMyPhone"}
+Location iPhone_Location                  "Coordinates"                                  (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:location"}
+Number   iPhone_LocationAccuracy          "Coordinates Accuracy [%.0f m]"                (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:locationAccuracy"}
+DateTime iPhone_LocationLastUpdate        "Last Update [%1$td.%1$tm.%1$tY, %1$tH:%1$tM]" (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:locationLastUpdate"}
+```
```

Comment:
From what I've understood here we should be good: https://www.apple.com/legal/intellectual-property/guidelinesfor3rdparties.html

Still as it doesn't hurt I'd add the following as a last sentence:

>Apple, iPhone, and iCloud are registered trademarks of Apple Inc., registered in the U.S. and other countries.

@kaikreuzer your opinion?

Reply:
We do not have this anywhere else (and it would probably be applicable similarly to other trademarks in 95% of the bindings) and I think Open Source projects are generally ok to live without it as there is no commercial use involved.
But as Apple might be special, I am ok to have that sentence here...

Reply:
"Apple might be special" was EXACTLY my thought 😅